README.md

```diff
@@ -119,6 +119,7 @@ In addition, integration modules are available for the following libraries, prov
 * [OptGBM: A scikit-learn compatible LightGBM estimator with Optuna](https://github.com/Y-oHr-N/OptGBM)
 * [PyKEEN](https://github.com/pykeen/pykeen)
 * [RL Baselines Zoo](https://github.com/DLR-RM/rl-baselines3-zoo)
+* [Hyperparameter Optimization for Machine Learning](https://github.com/solegalli/hyperparameter-optimization), online course repo
```

Comment:
How about using the same styles as the other entries?
More specifically, I propose two modifications:
- including the whole text to the link
- describe without abbreviation (i.e., replace repo with repository)

The following is just an example. Please feel free to update it.

```suggestion
* [Hyperparameter Optimization for Machine Learning, code repository for online course](https://github.com/solegalli/hyperparameter-optimization)
```